resolwe/flow/managers/__init__.py

```diff
@@ -37,5 +37,6 @@ def load_manager(manager_name):
             raise
 
 
-FLOW_MANAGER = getattr(settings, 'FLOW_MANAGER', 'resolwe.flow.managers.local')
-manager = load_manager(FLOW_MANAGER).Manager()  # pylint: disable=invalid-name
+FLOW_MANAGER = getattr(settings, 'FLOW_MANAGER', {}).get('NAME', 'resolwe.flow.managers.local')
+manager = load_manager(FLOW_MANAGER).Manager(static=True)  # pylint: disable=invalid-name
+MANAGER_CLASS = manager.__class__
```

Comment:
What is the reason for this? To avoid writing `manager.__class__`? Also, to be consistent with `FLOW_MANAGER` above it should probably be `FLOW_MANAGER_CLASS`?

Reply:
I got rid of this, it wasn't used anywhere in the end.